tool/tctl/common/user_command.go

```diff
@@ -135,7 +137,7 @@ func (u *UserCommand) ResetPassword(client auth.ClientI) error {
 func (u *UserCommand) PrintResetPasswordToken(token services.ResetPasswordToken, format string) error {
 	err := u.printResetPasswordToken(token,
 		format,
-		"Reset password token has been created and is valid for %v. Share this URL with the user to complete password reset process:\n%v\n\n",
+		"User %v has been reset. Share this URL with the user to complete password reset, link is valid for %v:\n%v\n\n",
```

Comment:
@benarent are you sure you want this message as opposed to explicitly mentioning reset password token?

Reply:
I'm happy with the solution, it focuses on the UX of the user being reset, with the outcome being a that they need to complete a reset password, vs being the other way around.